### DISCUSSION TITLE
Use library as an object within a node app?

### DISCUSSION THREAD

**Discussion by Hollywood** [apideck-libraries/portman#444] [Q&A]
I understand that this is meant to be run as a CLI utility, but is it possible to create a Portman object from within a node app and utilize its functionality that way?

My goal is to create an API endpoint that would allow me to send a collection to it and return the collection with different types of tests. The only part I'm stuck on is the question posted above.

Thanks a ton for any help! ✨

**Comment by thim81** [comment]
hi @Hollywood ,

Without really digging in, would it already be sufficient that you have the generated file in a location that you could read and return in the API?
